apollos-church-api/package.json

```diff
@@ -45,12 +45,13 @@
     "graphql": "14.6.0",
     "prettier": "1.12.1",
     "randomcolor": "0.5.3",
-    "sqlite3": "^5.0.1"
+    "sqlite3": "^5.0.1",
+    "umzug": "^3.0.0-beta.15"
```

Comment:
what's this for?

Reply:
Postgres migrations.

Reply:
why do we need it listed here when it's also in the `data-connetor-postgres` deps? i suppose this is a template question

Reply:
Yah, likely a template question. We can probably remove it from `templates`